pyramid/session.py

```diff
@@ -215,7 +215,7 @@ def check_csrf_token(request,
     supplied by ``request.session.get_csrf_token()``, and ``raises`` is
     ``True``, this function will raise an
     :exc:`pyramid.exceptions.BadCSRFToken` exception.
-    If the check does succeed and ``raises`` is ``False``, this
+    If the check does not succeed and ``raises`` is ``False``, this
```

Comment:
@mmerickel which one: mine or @dairiki's?

I think using "succeed" is difficult to parse due to double negatives and ambiguity of the context. Does "succeed" mean that the function returns `True` or that the check for matching tokens is `True`?